trestle/utils/fs.py

```diff
@@ -126,8 +114,7 @@ def get_contextual_model_type(path: pathlib.Path = None) -> Tuple[Type[OscalBase
     full_alias = model_alias
 
     for i in range(len(model_relative_path.parts)):
```

Comment:
I would rewrite this because of the range(len) and because the inner loop is somewhat opaque

Reply:
Cleaned up partially, however, it's inherited and a bit messy still. For not it is good enough for me.